python/looker_sdk/sdk/api40/methods.py

```diff
@@ -112,20 +112,9 @@ def search_alerts(
             self.get(
                 path="/alerts/search",
                 structure=Sequence[mdls.Alert],
-                query_params={
-                    "limit": limit,
-                    "offset": offset,
-                    "group_by": group_by,
-                    "fields": fields,
-                    "disabled": disabled,
-                    "frequency": frequency,
-                    "condition_met": condition_met,
-                    "last_run_start": last_run_start,
-                    "last_run_end": last_run_end,
-                    "all_owners": all_owners,
-                },
-                transport_options=transport_options,
-            ),
+                query_params={"limit": limit, "offset": offset, "group_by": group_by, "fields": fields, "disabled": disabled, "frequency": frequency, "condition_met": condition_met, "last_run_start": last_run_start, "last_run_end": last_run_end, "all_owners": all_owners},
+                transport_options=transport_options
+            )
```

Comment:
@jeremytchang @jkaster Here is another typical issue with newlines and commas after the last element in a list.

Reply:
Resolved with `pipenv run black api40/*.py`